### PR TITLE
Add labanquepostale.fr

### LINKED ISSUE #2258: Add labanquepostale.fr

### PR DESCRIPTION
**Pull request by divayprakash**
Closes #2258

### CODE REVIEW

File: _data/banking.yml [hunk 1]
```diff
@@ -381,6 +381,13 @@ websites:
       hardware: Yes
       doc: http://www.intesasanpaolo.com/servizi-multicanale/sicurezza.jsp
 
+    - name: La Banque Postale
+      url: https://www.labanquepostale.fr/
+      twitter: labanquepostale
```
Comment (Jawshy) [inline _data/banking.yml]: Please use the same case that their Twitter handle uses: [`LaBanquePostale`](https://twitter.com/labanquepostale).

**Review by Jawshy — CHANGES_REQUESTED** [review]

File: _data/banking.yml [hunk 2]
```diff
@@ -381,6 +381,13 @@ websites:
...
+      facebook: labanquepostale
+      img: labanquepostale.png
+      tfa: No
+
```
Comment (fpigerre) [inline _data/banking.yml:391]: Hello @divayprakash,
Would you also be able to add the following line: `lang: fr` please?

**Review by fpigerre — CHANGES_REQUESTED** [review]

File: _data/banking.yml (hunk 2 above)
Comment (divayprakash) [reply _data/banking.yml:391]: @psgs Added in d368469

File: _data/banking.yml (hunk 1 above)
Comment (divayprakash) [reply _data/banking.yml]: @Jawshy Updated in d368469

Comment (Jawshy) [reply _data/banking.yml]: You changed the `name` instead of the `twitter`.

Comment (divayprakash) [reply _data/banking.yml]: Fixed in 5c44a6c

**Review by Jawshy — APPROVED** [review]

**Review by fpigerre — APPROVED** [review]

**Comment by fpigerre** [comment]
Thanks for the pull request @divayprakash! 👍

**Pull request merged**